R/utilities-figures.R

```diff
@@ -482,7 +482,7 @@ createPlotsFromExcel <- function(simulatedScenarios, observedData, projectConfig
   }
   # Remove white spaces
   dfPlotGrids$plotIDs <- gsub(dfPlotGrids$plotIDs, pattern = " ", replacement = "", fixed = TRUE)
-  dfPlotGrids$plotIDs <- strsplit(dfPlotGrids$plotIDs, split = "[[:punct:]]")
+  dfPlotGrids$plotIDs <- strsplit(dfPlotGrids$plotIDs, split = ",")
```

Comment:
@hannaei We should only use `,`  as separator. Otherwise, plot ids like `P_1` are split into `P` and `1`.